be/src/runtime/exec_env.cpp

```diff
@@ -789,13 +789,29 @@ Status ExecEnv::init(const std::vector<StorePath>& store_paths, bool as_cn) {
     }
     setenv(staros::starlet::fslib::kFslibCacheDir.c_str(), config::starlet_cache_dir.c_str(), 1);
 
+    std::unique_ptr<ThreadPool> put_aggregate_metadata_thread_pool;
+    int32_t max_thread_count = config::transaction_publish_version_worker_count;
```

Comment:
consider make this configvar dynamic changeable so it will be much easier to update the configvar and take effect without restarting the BE process.

Reply:
> consider make this configvar dynamic changeable so it will be much easier to update the configvar and take effect without restarting the BE process.

done.